### PR TITLE
Update digital_micrograph.py

### PR DESCRIPTION
**Pull request by erh3cq**
Add support for microscope name stored under Microscope_Info.Microscope
Add support for aperture_size when EELS.aperture_size is '2mm' or '2 mm'

### CODE REVIEW

File: hyperspy/io_plugins/digital_micrograph.py [hunk 1]
```diff
@@ -922,7 +924,7 @@ def get_mapping(self):
                     None),
                 "ImageList.TagGroup0.ImageTags.EELS_Spectrometer.Aperture_label": (
                     "Acquisition_instrument.TEM.Detector.EELS.aperture_size",
-                    lambda string: float(string.replace(' mm', ''))),
+                    lambda string: float(string.replace('mm', '').replace(' ',''))),
```
Comment (francisco-dlp) [inline hyperspy/io_plugins/digital_micrograph.py]: Maybe just the following would do?

```python

lambda string: float(string.replace('mm', '')),

```

**Comment by francisco-dlp** [comment]
If you are quick at replying to the little comment above this may make it into v1.2 (due today). By the way, that will add you to the authors in our zenodo doi but, unless you update your name and affiliation in your github profile now, you'll appear as erh3cq.

File: hyperspy/io_plugins/digital_micrograph.py (hunk 1 above)
Comment (erh3cq) [reply hyperspy/io_plugins/digital_micrograph.py]: then it leaves the string as '2 ' and i think it needs '2'?

**Comment by erh3cq** [comment]
Updated. Thank you!

File: hyperspy/io_plugins/digital_micrograph.py (hunk 1 above)
Comment (francisco-dlp) [reply hyperspy/io_plugins/digital_micrograph.py]: I think that ``float`` wouldn't care about spaces.

Comment (erh3cq) [reply hyperspy/io_plugins/digital_micrograph.py]: yes you are correct.  How do I update this pull (just starting with gitHub)

**Pull request merged**